CHANGELOG.md

```diff
@@ -4,9 +4,27 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Next
+## 0.5.10
 
-* Fixed warning that `std::mem::forget` was called a reference while it should receive an owned object.
+### Highlights
+
+This release brings:
+- You can now pass hexadecimal strings with the `0x` prefix to the `Hex::decode` function. Hexadecimal strings without the prefix are still supported.
+- Fixed warning that `std::mem::forget` was called a reference while it should receive an owned object.
```

Comment:
I think we can leave that one, it's internal to the project, I'll merge and remove it for now